File: C++/Programming Basics with C++/05. Exercise - ConditionalStatement/simple_tasks/WorldSwimmingRecord.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double worldRecord, distance, speed;
    cin >> worldRecord >> distance >> speed;

    double delay = floor(distance / 15) * 12.5;
    double totalTime = distance * speed + delay;

    cout.setf(iostream::fixed);
    cout.precision(2);

    if (worldRecord > totalTime) {
        cout << "Yes, he succeeded! The new world record is " << totalTime << " seconds." << endl;
    } else {
        cout << "No, he failed! He was " << totalTime - worldRecord << " seconds slower." << endl;
    }

    return 0;
}

File: C#/Programming Basics with C#/05. Conditional Statements - Exercise/08. World Swimming Record/WorldSwimmingRecord.cs
using System;

class WorldSwimmingRecord
{
    static void Main()
    {
        double WorldSwimmingRecord = double.Parse(Console.ReadLine());
        double DistanceMeters = double.Parse(Console.ReadLine());
        double IvanTimeFotMeter = double.Parse(Console.ReadLine()); //seconds
        double IvanSwimingTime = DistanceMeters * IvanTimeFotMeter + Math.Floor(DistanceMeters / 15.0) * 12.5;
        if (WorldSwimmingRecord > IvanSwimingTime)
        {
            Console.WriteLine($"Yes, he succeeded! The new world record is {IvanSwimingTime:F2} seconds.");
        }
        else
        {
            Console.WriteLine($"No, he failed! He was {(IvanSwimingTime - WorldSwimmingRecord):F2} seconds slower.");
        }
    }
}